src/query-builder/delete-query-builder.ts

```diff
@@ -716,6 +716,66 @@ export class DeleteQueryBuilder<DB, TB extends keyof DB, O>
     })
   }
 
+  /**
+   * This can be used to add any additional SQL to the front of the query __after__ the `delete` keyword.
```

Comment:
This is not aligned with the example, as the additional SQL is not added **right after the keyword**. Confusing.

Reply:
will remove it